closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -57,6 +57,8 @@ public final class RestrictedSecurity {
     private static boolean isFIPSSupported;
     private static boolean isFIPSEnabled;
 
+    private static final boolean allowSetSecurityProps;
```

Comment:
Suggest `allowSetProperties` is a better name; it more closely matches the name of the property from which it is derived.

While you're here, please correct the comment on line 55 (this applies to more than just Linux x64).

Reply:
Changed both.